### PR TITLE
Issue 3 edit entry

### PR DESCRIPTION
**Pull request by gregg-garceau**
https://user-images.githubusercontent.com/88458577/132757975-7bc6db80-5746-48a5-aa18-7eccda4b506c.mp4

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -40,44 +47,74 @@ function renderEntries(entry) {
...
+
+    var prependEntry = renderEntries(entryForm);
+    $unorderedList.prepend(prependEntry);
+  }
+  data.editing = null;
   $dataViewEntries.className = 'view';
   $dataViewForm.className = 'hidden';
   data.view = 'entries';
-
+  location.reload();
```
Comment (scbowler) [inline js/main.js]: Using `location.reload()` is not the way to handle updating your entries list. Refreshing the page unexpectedly is bad practice and makes for a bad user experience. You need to use the new or updated entry to update the DOM directly.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, you just need to update the dom without refreshing the page. Let us know if you have any questions!

File: js/main.js (hunk 1 above)
Comment (gregg-garceau) [reply js/main.js]: https://user-images.githubusercontent.com/88458577/132767921-4b864381-9a71-4401-a99b-c91061d81ce8.mp4

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**